src/main/java/seedu/algobase/logic/parser/EditCommandParser.java

```diff
@@ -33,7 +36,7 @@ public EditCommand parse(String args) throws ParseException {
         requireNonNull(args);
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_AUTHOR, PREFIX_WEBLINK, PREFIX_DESCRIPTION,
-                        PREFIX_TAG);
+                        PREFIX_TAG, PREFIX_DIFFICULTY, PREFIX_REMARK, PREFIX_SOURCE);
```

Comment:
Since there are additional prefixes now, we should add the checks for `difficulty`, `remark` and `source` to line 63 following the same styles as lines 50 to 62